src/app/login/login.component.html

```diff
@@ -0,0 +1,46 @@
+<mat-card class= "background" >
+  <div class ="color-back">
+      <div class ="icon">
+        <span style ="font-size:60px;"> 
```

Comment:
please don't use inline css. move it in a css class

Reply:
removed